### PR TITLE
Known null safety issues

### PR DESCRIPTION
**Pull request by mit-mit**
Add a new section with known null safety issues.

Corresponding Dart PR: https://github.com/dart-lang/site-www/pull/2948

### CODE REVIEW

File: src/docs/null-safety.md [hunk 1]
```diff
@@ -54,6 +56,22 @@ check out these pages:
   in which only some libraries are null safe.
 * [FAQ][]:
   Questions that have come up during migration to null safety.
+  
+## Known issues
+
+Not all parts of the Flutter SDK support null safety yet,
+as some parts still need additional work to
+[migrate to null safety](https://dart.dev/null-safety/migration-guide).
```
Comment (sfshaza2) [inline src/docs/null-safety.md:64]:
```suggestion
[migrate to null safety]({{site.dart-site}}/null-safety/migration-guide).
```

File: src/docs/null-safety.md [hunk 2]
```diff
@@ -54,6 +56,22 @@ check out these pages:
...
+
+We're currently aware of the following issues:
+
+  * Migration of the pub.dev packages owned by the Flutter team
+    is in progress. See pub.dev for
+    [the current list](https://pub.dev/packages?q=publisher%3Aflutter.dev&null-safe=1).
```
Comment (sfshaza2) [inline src/docs/null-safety.md:70]:
```suggestion
    [the current list]({{site.pub}}/packages?q=publisher%3Aflutter.dev&null-safe=1).
```

File: src/docs/null-safety.md [hunk 3]
```diff
@@ -54,6 +56,22 @@ check out these pages:
...
+
+  * Integration testing with
+    [`flutter_driver`](https://flutter.dev/docs/cookbook/testing/integration/introduction)
```
Comment (sfshaza2) [inline src/docs/null-safety.md:73]:
```suggestion
    [`flutter_driver`](/docs/cookbook/testing/integration/introduction)
```

**Review by sfshaza2 — APPROVED** [review]
LGTM!

File: src/docs/null-safety.md [hunk 4]
```diff
@@ -54,6 +56,22 @@ check out these pages:
...
 * [FAQ][]:
   Questions that have come up during migration to null safety.
+  
+## Known issues
+
+Not all parts of the Flutter SDK support null safety yet,
+as some parts still need additional work to
+[migrate to null safety]({{site.dart-site}}/null-safety/migration-guide).
+
+We're currently aware of the following issues:
```
Comment (jakemac53) [inline src/docs/null-safety.md:66]: Should we specify that even depending on `flutter_driver` may limit you from picking up other dependencies that have migrated?

**Comment by sfshaza2** [comment]
I'm happy to do that in a suggestion, because it does seem important.

File: src/docs/null-safety.md [hunk 5]
```diff
@@ -54,6 +56,22 @@ check out these pages:
...
+
+  * Migration of the pub.dev packages owned by the Flutter team
+    is in progress. See pub.dev for
+    [the current list]({{site.pub}}/packages?q=publisher%3Aflutter.dev&null-safe=1).
+
+  * Integration testing with
+    [`flutter_driver`](/docs/cookbook/testing/integration/introduction)
+    currently does not support null safety.
```
Comment (jakemac53) [inline src/docs/null-safety.md:74]:
```suggestion
    currently does not support null safety.
    
  * Depending on `flutter_driver` will limit your ability to pick up some
    already migrated dependencies such as `args`, `archive`, and `crypto`.
```

cc @kevmoo @natebosch ptal?

File: src/docs/null-safety.md (hunk 4 above)
Comment (mit-mit) [reply src/docs/null-safety.md:66]: Thanks, took your change below

**Comment by kwalrath** [comment]
I plan to have a followup PR that updates this page to reflect any changes we decide to make in dart-lang/site-www#2948.

**Comment by sfshaza2** [comment]
sg

**Pull request merged**